bitrise.yml

```diff
@@ -81,16 +83,16 @@ workflows:
     - certificate-and-profile-installer@1: {}
     - cocoapods-install@1:
         inputs:
-        - podfile_path: "$BITRISE_POD_FILE_PATH"
+        - podfile_path: $BITRISE_POD_FILE_PATH_OLD
```

Comment:
should we keep this as BITRISE_POD_FILE_PATT instead of OLD ?

Reply:
I've cleaned up all the constants (there was a lot of duplication and mess in there)
See [PI-XXXX: cleanup constants](https://github.com/checkout/frames-ios/pull/147/commits/0282f0665b8f22b7094649dd59b1cb994898d1ad)